### ISSUE TITLE
Produce HTML from kdrfc

### ISSUE THREAD

**Issue by martinthomson** [cabo/kramdown-rfc2629#45]
I need this for martinthomson/i-d-template#106

**Comment by cabo** [comment]
Right.  I need to examine passing through more options to xml2rfc (including -u, for instance).  Obviously, I need to translate all the options to POST parameters for -r.

So the design would be: You can add any number of --text, --html (and --nroff, --raw, --exp, ... even a --xml to suppress the xml2rfc call), generating the appropriate number of xml2rfc calls.  If you supply none of those, you get --text (as opposed to nothing).  You get XML output (the equivalent of --xml) in any case; --xml is just there to suppress the --text default.

**Comment by martinthomson** [comment]
I'd add `-u` unconditionally.  But then I submit XML to the tracker.  The text/html versions are used for preview purposes only and there making a sane choice (i.e., Unicode) doesn't have negative consequences.
